# Project: HTML Forms

yigitm opened the pull request:
In this project; we worked with HTML form elements to design a sign-up page with below main features.

-Some media queries are used for different resolutions & screen sizes.
-CSS Flex boxes are used for better alignment purposes.
-All HTML & CSS errors are checked.

Thanks in advance for your time & energy to improve the project.

Dipeshtwis committed:
Add nav-bar section

yigitm committed:
Add form and elements

Dipeshtwis committed:
Add form elements and form footer with CSS

yigitm committed:
Add nav-bar and media queries

Dipeshtwis committed:
Add footer section and image in button

Dipeshtwis committed:
Validate Html and CSS with no error

yigitm committed:
Add REQADME.md file

lucasmazo32 commented:
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved kindly implement these changes:

## Required changes:

- [x] Be careful with the type of input that you are using for the password! You don't want to be sharing your screen and other people seeing your passwords.

- [x] Please enable the stickler for this repo. You can enable your stickler [here](https://stickler-ci.com/). Remember that you need to make at least one commit to see if it works. In case you are having problems, you can always check this microverse [documentation](https://github.com/microverseinc/linters-config/tree/master/css).

### Happy coding! :computer:

## Lucas Mazo

yigitm committed:
Update password input type

Dipeshtwis committed:
Align form and its element to center

Dipeshtwis committed:
Change the font-style

yigitm committed:
Update logo and link sections

yigitm committed:
Update readme file screenshot

Dipeshtwis committed:
Validate CSS and Html and add link to the icon and text

Dipeshtwis commented:
Update in this project includes

- I have used the right input type for password.
- Allow stickler to check our repository.
- made separate icons that are associated with links.
- Aligned form elements.

Thank you in advance for your time and energy for review.

santiago-dev96 commented:
## Pull Request under review by @santiago-rodrig (TSE)

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+
+.nav-flex {
+ display: flex;
+ flex-direction: row;
```
Flex boxes have a direction of row by default.
```suggestion
```

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ width: 100%;
```
Flex boxes take 100% width of their parent elements.
```suggestion
```

yigitm replied:
Thank you so much. We are proceeding with each of these changes as suggested. This one is done.

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+ justify-content: flex-end;
+ width: 100%;
+ height: 200px;
+ vertical-align: middle;
```
I guess that what you wanted is to center the items inside the flex box vertically, because `vertical-align` does something totally different (take a read [here](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align)).
```suggestion
  align-items: center;
```

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+}
+.flex-sub-logo {
+ display: flex;
+ flex-direction: row;
```
```suggestion
```

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+.flex-sub-logo {
+ display: flex;
+ flex-direction: row;
+ flex-flow: row;
```
This property is not being properly used, take a read [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow).
```suggestion
```

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+ flex-direction: row;
+ flex-flow: row;
+ width: 100%;
+ height: auto;
```
This is the default.
```suggestion
```

Dipeshtwis replied:
CSS updated.

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ font-size: 12px;
+ font-weight: lighter;
+ text-align: left;
```
This is the default.
```suggestion
```

santiago-dev96 commented on style.css:
```diff
@@ -1 +1,263 @@
+ width: 100%;
+}
+
+.account-info input[type="email"] {
+ padding-left: 10px;
+ padding-bottom: 8px;
+ margin-bottom: 6px;
+}
+.account-info input[type="tel"] {
+ padding-bottom: 8px;
+ padding-left: 10px;
+}
+.account-info input[type="password"] {
+ padding-bottom: 8px;
+ padding-left: 10px;
+}
```
You are repeating yourself.
```suggestion
.account-info input[type="email"],
.account-info input[type="tel"]
.account-info input[type="password"] {
  padding-bottom: 8px;
  padding-left: 10px;
}

.account-info input[type="email"] { margin-bottom: 6px; }
```

santiago-dev96 reviewed (approved):
# Project approved! :100:
I left some suggestions, read them if you want, it will help you in future projects. Have a nice day :wave:!

yigitm committed:
Update style.css

Thank you so much. We are proceeding with each of these changes as suggested. This one is done.

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

Css updated

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

Css updated

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

css updated

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

CSS updated

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

CSS updated.

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm committed:
Update style.css

CSS updated

Co-authored-by: Santiago Rodríguez <santo1996.29@gmail.com>

yigitm commented:
All suggested changes are done.

yigitm committed:
CSS updated

yigitm committed:
CSS updated

yigitm committed:
CSS updated - stickler
